modules/db/src/test/scala/gem/dao/check/Check.scala

```diff
@@ -0,0 +1,64 @@
+package gem.dao
+package check
+
+import gem._
+import gem.enum._
+import gem.config._
+import edu.gemini.spModel.core.{ Site, ProgramType, Semester, _ }
+
+import doobie.imports._
+import doobie.scalatest.imports._
+import org.scalatest._
+
+import scalaz._, Scalaz._
+
+/** Trait for tests that check statement syntax and mappings. */
+trait Check extends FlatSpec with Matchers with QueryChecker {
```

Comment:
Yeah it's just a trait that provides a utility method to create schema checks and print out errors in a nice way. Not much to it.